icu4j/main/collate/src/test/java/com/ibm/icu/dev/test/collator/UCAConformanceTest.java

```diff
@@ -181,6 +181,7 @@ private static int normalizeResult(int result) {
         return result < 0 ? -1 : result == 0 ? 0 : 1;
     }
 
+    @SuppressWarnings("ReferenceEquality")
```

Comment:
still needed after changing to .equals()?

Reply:
Yes, because in line 271 we have this:
```java
if (oldSk == sk1) {
    newSk = sk2;
} else {
    newSk = sk1;
}
```
Not a fun, but...

One of the things I don't like about `@SuppressWarnings` is that (in most cases) you can't just dump it in the middle of a block of code.
So you put it at the top of a method, to counter something that happens 100 lines down.

And replacing it with `.equals` fails.
I can debug it to understand why.
But it kind of shows why this kind of behavior is tricky.